cli/src/commands/create-mint/index.ts

```diff
@@ -0,0 +1,71 @@
+import { Args, Command, Flags } from "@oclif/core";
+import {
+  CustomLoader,
+  getPayer,
+  getRpc,
+  getSolanaRpcUrl,
+} from "../../utils/utils";
+import { createMint } from "@lightprotocol/compressed-token";
+import {
+  ConfirmOptions,
+  Connection,
+  Keypair,
+  PublicKey,
+  Signer,
+} from "@solana/web3.js";
+import { confirmTx } from "@lightprotocol/stateless.js";
+class CreateMintCommand extends Command {
+  //light create-mint -mint Optional<Keypair> --mint-decimals <u8>
+  static summary = "Create a new compressed token mint";
+
+  static examples = ["$ light create-mint --mint-decimals 2"];
+
+  static flags = {
+    // ...standardFlags,
+    // ...confirmOptionsFlags,
+    "mint-decimals": Flags.integer({
+      description: "Mint decimals.",
+      required: false,
+      default: 2,
+    }),
+  };
+
+  static args = {};
+
+  async run() {
+    const { args, flags } = await this.parse(CreateMintCommand);
+
+    const loader = new CustomLoader(`Performing create-mint...\n`);
+    loader.start();
+
+    try {
+      let mintDecimals = flags["mintDecimals"];
+      if (!mintDecimals) {
+        mintDecimals = 2;
+      }
+      const payer = getPayer();
+
+      const randomMint = Keypair.generate();
```

Comment:
for users who don't want to pass in a specific mint keypair into cmd args, `createMint() `already creates a random mint keypair on the fly under the hood, so you can just pass `undefined`